assignments/lambda-classes.js

```diff
@@ -1 +1,154 @@
 // CODE here for your Lambda Classes
+class Person {
+  constructor(personAttr){
+    this.name = personAttr.name
+    this.age = personAttr.age
+    this.location = personAttr.location
+    this.gender = personAttr.name
```

Comment:
Did you mean for this to be `this.gender = personAttr.gender`?

Reply:
Ah you're right! Thank you.